.changeset/dirty-pandas-pay.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': major
+'@firebase/firestore': major
+---
+
+Added support for FieldValues when using a FirestoreDataConverter and support for update() calls with a converter
```

Comment:
This needs to be much more elaborate and explain what is breaking, how to use it and other things. This will be the only mention of this.

Reply:
Updated.